editor/editor_node.cpp

```diff
@@ -4339,6 +4339,15 @@ bool EditorNode::ensure_main_scene(bool p_from_native) {
 	return true;
 }
 
+void EditorNode::run_play() {
+	_menu_option_confirm(RUN_STOP, true);
+	_run(false);
+}
+
+void EditorNode::run_stop() {
+	_menu_option_confirm(RUN_STOP, false);
+}
+
```

Comment:
I had to expose this to allow running and stopping the game programmatically from the editor.

Reply:
Can existing EditorNode code be changed to use those new public methods, for consistency?

Reply:
The `run_stop` logic is actually in the `_menu_option_confirm` switch. Not sure if it would be a good idea to move that code to this function and the switch call it instead, I'm not familiar with this.